onnx/optimizer/passes/eliminate_identity.h

```diff
@@ -0,0 +1,30 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE { namespace optimization {
+
+struct EliminateIdentity final : public OptimizePass {
+  explicit EliminateIdentity()
+    : OptimizePass("eliminate_identity", API_TYPE::IR) {
+  }
+
+  void eliminate_identity(Graph& graph) {
+    for (auto it = graph.begin(); it != graph.end(); ++it) {
+      auto* n = *it;
+      if (n->kind() == kIdentity) {
+        n->output()->replaceAllUsesWith(n->input());
```

Comment:
If an Identity is connected with a graph output, we should keep the output name?

Reply:
For now, I think no need, because we didn't keep the output name in other passes either.

But it may be worth to create some helper functions to rename the inputs and outputs